helix-term/src/application.rs

```diff
@@ -569,12 +569,15 @@ impl Application {
         }
 
         // TODO: fix being overwritten by lsp
-        self.editor.set_status(format!(
-            "'{}' written, {}L {}B",
-            get_relative_path(&doc_save_event.path).to_string_lossy(),
-            lines,
-            bytes
-        ));
+        match get_relative_path(&doc_save_event.path) {
+            Ok(path) => self.editor.set_status(format!(
+                "'{}' written, {}L {}B",
+                path.to_string_lossy(),
+                lines,
+                bytes
+            )),
+            _ => self.editor.set_error("Error trying to save file"),
+        }
```

Comment:
This isn't actually an error saving though, just an error formatting this path as relative. Instead of erroring here let's use the full path if we can't determine the relative one:

```suggestion
        let path =
            get_relative_path(&doc_save_event.path).unwrap_or_else(|_| doc_save_event.path.into());
        self.editor.set_status(format!(
            "'{}' written, {}L {}B",
            path.to_string_lossy(),
            lines,
            bytes
        ));
```